linode/manager
# M3-5376 Volume create, delete, attach, clone, and resize flows

## Description

**What does this PR do?**
Adds Cypress E2E tests for volume creation, deletion, attach/detach, clone, and resize flows using real data.

There are still some flows that aren't covered (for example, clone and resize via Linode details pages) but I'm hoping this at least puts us in better shape, and that we can improve coverage for other flows with time.

This PR also includes some reorganization of the existing Cypress utility functions, and includes some new utilities to facilitate the use of the `api-v4` package for making API requests. In future PRs I'll begin removing some of the API functions in `cypress/support/api` in favor of their `api-v4` counterparts.

## How to test
The following command can run the new/modified E2E tests to ensure they work as expected:
```bash
yarn cy:e2e --spec "cypress/integration/volumes/*"
```

If you want to confirm that no existing E2E tests were broken by these changes, you can run them all:
```bash
yarn cy:e2e
```
